tools/build_release_uvision.bat

```diff
@@ -15,22 +15,40 @@
 :: See the License for the specific language governing permissions and
 :: limitations under the License.
 ::
+
```

Comment:
If you change to the root directory of the project at the start of the script then you can run this from anywhere.  To do this "cd %~dp0..\" would be needed here.